README.md

```diff
@@ -188,6 +188,75 @@ TODO:
 - broadcasting responses??
 - ??
 
+### Wants
+
+Wants packets are ... TODO
+
+A want packet is encoded as the concatenation of:
+
+section   | bytes                | description
+:---------|:---------------------|:------------
+`dmx`     | 7                    | identifies the feeds you're coordinating around
+`payload` | <= 113               | bipf encoded data
+`padding` | 113 - payload.length | topping the packet up to 120 Bytes
+
+```
+  |<----------------120------------------>|
+
+     7             x              113-x
+  +-----+----------------------+----------+
+  | dmx | payload              | padding? |
+  +-----+----------------------+----------+
+```
+
+#### Want `dmx`
+
+`dmx` is the calculated the first 7 Bytes of the sha256 hash of the string
+`"want"` (encoded as :fire: TODO), concatenated with the ordered set of
+`feed_ids` you are coordinating around. 
+
+In pseudocode:
+
+```
+dmx_material = 'want' + feed_1_id + ... + feed_N_id
```

Comment:
Nanomonkey:
- has `dmx_prefix`
- `"want"` is encoded as ....
- goset state is 32 bytes of ... XOR...